packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -57,7 +58,7 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
   const availableLocations = useLocations();
   const { data: availablePrograms } = useAvailablePrograms();
   const { data: enrollments, mutateEnrollments } = useEnrollments(patientUuid);
-  const { showProgramStatusField } = useConfig();
+  const { showProgramStatusField, allowCompletionTime } = useConfig();
```

Comment:
```suggestion
  const { showProgramStatusField, allowCompletionTime } = useConfig<ConfigObject>();
```

So, this `useConfig` hook ought to be annotated with the [ConfigObject interface] (https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-programs-app/src/config-schema.ts#L16) from the Programs app's config schema. When we do that, it'll flag that `allowCompletionDate` is not a known property of the config schema and that it should be added to the schema first before referencing it here.

Reply:
Thanks @denniskigen